go/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.8
+
+WORKDIR /go/src/github.com/Originate/exocom/go/
+COPY . .
+
+WORKDIR exocom
+RUN go install
+RUN rm -rf /go/src/
+
+CMD /go/bin/exocom
```

Comment:
Copying the entire go folder to get the vendor and shared (util / struct) directories. Build the binary, then remove all source to make the docker image smaller. Finally give it the default command as bin/exocom and thus we don't need to supply a command on exosphere to run exocom